lib/features/thread/presentation/thread_controller.dart

```diff
@@ -316,10 +316,7 @@ class ThreadController extends BaseController with EmailActionController {
 
     ever(mailboxDashBoardController.emailUIAction, (action) {
       if (action is RefreshChangeEmailAction) {
-        if (action.newState != _currentEmailState) {
```

Comment:
This check actually had value. It's the point of https://github.com/linagora/tmail-flutter/issues/3331 : ignore websocket updates we recently generated.

We could keep a cache (First in first out with fixed size of 128 items) of the newState generated by our action and challenge incoming websocket stateChange against it?

Reply:
Of course. We have put that handling inside the refreshChange function. Please see the code in that function for more details.